src/components/Title/index.js

```diff
@@ -314,22 +336,57 @@ class Title extends PureComponent /*:: <Props> */ {
         )}
 
         <div className={f('title-name')}>
-          {
-            // add margin only for IPSCAN result page
-            metadata.name.name === 'InterProScan Search Result' ? (
-              <h3 className={f('margin-bottom-large')}>
-                {metadata.name.name}{' '}
-              </h3>
-            ) : (
-              <h3>{metadata.name.name}</h3>
-            )
-          }
+          <div className={f('title-fav')}>
+            {
+              // add margin only for IPSCAN result page
+              metadata.name.name === 'InterProScan Search Result' ? (
+                <h3 className={f('margin-bottom-large')}>
+                  {metadata.name.name}{' '}
+                </h3>
+              ) : (
+                <h3>{metadata.name.name}</h3>
+              )
+            }
+            {
+              // Showing Favourites only for InterPro entries for now
+              isEntry &&
+                metadata.type &&
+                metadata.source_database &&
+                metadata.source_database.toLowerCase() === 'interpro' && (
+                  <span
+                    className={f('fav-icon')}
+                    role="button"
+                    onClick={() => this.manageFavourites(metadata)}
+                    onKeyDown={() => this.manageFavourites(metadata)}
+                    tabIndex={0}
+                  >
+                    <span
+                      className={f(
+                        'icon',
+                        'icon-common',
+                        this.props.entries.includes(metadata.accession)
```

Comment:
Changed this way as I think it's a bit more react-like...
this way the dom element doesn't change only its css class.

Although I'm not actually sure if the virtual DOM takes care of it

Reply:
Oh right. I was just getting rid of the errors and warnings at the time which I forgot to refactor it later.